code/modules/mob/living/living_powers.dm

```diff
@@ -32,10 +32,34 @@
 
 	attempt_devour(L, eat_types, mouth_size)
 
-/*
-/mob/living/verb/devourverb(var/mob/living/victim)//For situations where species inherent verbs isnt suitable
-	set category = "Abilities"
-	set name = "Devour Creature"
-	set desc = "Attempt to eat a nearby creature, swallowing it whole if small enough, or eating it piece by piece otherwise"
-	attempt_devour(victim, eat_types, mouth_size)
-*/
\ No newline at end of file
+/mob/living/verb/set_walk_speed()
```

Comment:
i dont see any way you could activate this menu on someone else's behalf, it doesnt seem like it'd affect anything